Sources/SnowManLogic/ViewLogic/MockIO.swift

```diff
@@ -0,0 +1,22 @@
+public class MockIO: Receivable {
+	public var stubbedUserInput: [String]!
+	var counter: Int
+
+	public init() {
+		counter = 0
+	}
+
+	public func getUserInput(getInput: () -> String) -> String {
+		return String(getInput())
+	}
+
+	public func myReadLine() -> String {
+		let value = stubbedUserInput[counter]
+		counter += 1
+		return value
+	}
+
+	public func display(linesToPrint: String) {
+		print(linesToPrint)
+	}
+}
```

Comment:
Your mock is too complicated, it should be void of an logic, just getters, setters, and functions that wrap return values.